git-remote-mediawiki.perl

```diff
@@ -962,7 +971,7 @@ sub mw_import_revids {
 		# If this is a revision of the media page for new version
 		# of a file do one common commit for both file and media page.
 		# Else do commit only for that page.
-		print {*STDERR} "${n}/", scalar(@{$revision_ids}), ": Revision #$rev->{revid} of $commit{title}\n";
+		print {*STDERR} "${n}/", scalar(@{$revision_ids}), ": Revision #$rev->{revid} of ", fe_escape_path($commit{title}), "\n";
```

Comment:
how's that related to subdir support?

Reply:
@anarcat It looks like I was trying to get it visualized better. This is just a user message, right?

Reply:
okay, let's keep it out of the patch then.